components/kyma-environment-broker/Dockerfile.scj

```diff
@@ -0,0 +1,25 @@
+FROM eu.gcr.io/kyma-project/external/golang:1.17.6-alpine3.15 as build
+
+WORKDIR /go/src/github.com/kyma-project/control-plane/components/kyma-environment-broker
+
+COPY cmd cmd
+COPY common common
+COPY internal internal
+COPY go.mod go.mod
+COPY go.sum go.sum
+COPY vendor vendor
+
+RUN CGO_ENABLED=0 go build -o /bin/subscriptioncleanup ./cmd/subscriptioncleanup/main.go
+
+# Get latest CA certs
+FROM alpine:latest as certs
+RUN apk --update add ca-certificates
+
+# Final image
+FROM alpine:3.15.0
+LABEL source = git@github.com:kyma-project/control-plane.git
+
+COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+COPY --from=build /bin/subscriptioncleanup /bin/subscriptioncleanup
+
```

Comment:
Please consider using WORKDIR /app convention as it in the previous SCJ Dockerfile

Reply:
Done.